confluent_kafka/avro/cached_schema_registry_client.py

```diff
@@ -338,10 +343,10 @@ def get_version(self, subject, avro_schema):
 
         result, code = self._send_request(url, method='POST', body=body)
         if code == 404:
-            log.error("Not found:" + str(code))
+            log.error("Schema for {} Not found. Error code: {}".format(subject, code))
```

Comment:
Should `N` in not be lower cased? 

How about `Schema for subject '{}' not found`?

Reply:
SGTM 👍